tests/greenheart/test_openmdao.py

```diff
@@ -379,3 +415,37 @@ def test_run_greenheart_optimize(subtests):
     
     with subtests.test("lcoh"):
         assert lcoh_final < lcoh_init
+
+# def test_greenheart_offshore_wind_wave_solar_battery():
```

Comment:
Do we want to keep these?

Reply:
I may put it back in complete later, but not needed for now.

Reply:
Removed